iup/__init__.py

```diff
@@ -1,7 +1,8 @@
-import polars as pl
 import datetime as dt
 from typing import Sequence
 
+import polars as pl
```

Comment:
Looks like this is just moved a few lines - any reason? Also, beware there will be conflicts here since the last PR #72 merged, so there are some further imports (e.g. `from typing import List` and `from polars.datatypes.classes import DataTypeClass`) necessary.

Reply:
The rearrangement is related to #76 , which I just opened to formalize this

I'll rebase and catch those other points!